src/Config.hs

```diff
@@ -119,6 +120,7 @@ defaultOptions = Options
   { oVaultHost      = Just "localhost"
   , oVaultPort      = Just 8200
   , oVaultAddr      = parseURI "https://localhost:8200"
+  , oVaultAuthBackend = Just ""
```

Comment:
I think this should be `Nothing`, otherwise we would be sending requests to `/v1/auth//login` instead of `/v1/auth/github/login` and `/v1/auth/kubernetes/login` by default.